base_layer/core/src/chain_storage/lmdb_db/mod.rs

```diff
@@ -72,3 +77,13 @@ pub(crate) struct TransactionKernelRowData {
     pub mmr_position: u32,
     pub hash: HashOutput,
 }
+
+hash_domain!(BaseLayerCoreDomain, "com.tari.tari-project.base_layer.core");
+
+pub(crate) const LMDB_STORAGE_HASH_LABEL: &str = "lmdb_db";
+
+pub(crate) fn base_layer_core_chain_storage_lmdb_hasher<D: Digest + LengthExtensionAttackResistant>(
```

Comment:
I think the requirement for `LengthExtensionAttackResistant1` should be in the function where the hasher is used.